zerver/data_import/slack.py

```diff
@@ -1366,10 +1387,15 @@ def fetch_shared_channel_users(
             bot_id = message["bot_id"]
             if bot_id in integration_bot_users:
                 continue
-            bot_info = get_slack_api_data(
-                "https://slack.com/api/bots.info", "bot", token=token, bot=bot_id
-            )
-            bot_user = convert_bot_info_to_slack_user(bot_info)
+            try:
+                bot_info = get_slack_api_data(
+                    "https://slack.com/api/bots.info", "bot", token=token, bot=bot_id
+                )
+            except SlackBotNotFoundError:
+                logging.info("Bot %s not found, making a dummy instead", bot_id)
```

Comment:
"dummy" implies `is_mirror_dummy` -- we should sayL
```suggestion
                logging.info("Bot %s not found, creating a deleted placeholder", bot_id)
```
...or similar